docs/docs/50-user-guide/20-how-to-guides/30-working-with-warehouses.md

```diff
@@ -671,6 +671,19 @@ Helm chart repository subscriptions can be defined using the following fields:
         semverConstraint: ^1.0.0
   ```
 
+
```

Comment:
Markdown linters usually doesn't like double line breaks -- at least the one I'm using does not.

```suggestion
```